internal/backend/conjur/templates.go

```diff
@@ -82,7 +82,9 @@ func (c StorageBackend) ListTemplates() ([]string, error) {
 func (c StorageBackend) GetTemplate(templateName string) (types.Template, error) {
 	variableID := c.getTemplateVariableID(templateName)
 	templateJSON, err := c.client.RetrieveSecret(variableID)
-	template := &types.Template{}
+	template := &types.Template{
+		StoreCertificate: true,
```

Comment:
Is this to set the default bool value to true?  Just wanted to confirm...

Reply:
It defaults to true, and if false is provided is overwrites the true.